### ISSUE TITLE
[VisUI] Tooltip$DisplayTask throws NPE if target actor was removed from the stage

### ISSUE THREAD

**Issue by metaphore** [kotcrab/vis-editor#241]
VisUI version: **1.2.4**. Here is how it happens:
1. `Tooltip$TooltipInputListener` is assigned to target actor to show `Tooltip` over it.
2. Hover mouse pointer over target actor and remove it from stage.
3. `TooltipInputListener` receives events in next order `enter`, `exit`, `enter`.
4. Upon last `enter` event `TooltipInputListener` pushes `DisplayTask` into `Timer`'s queue (which operates in separate thread)
5. When `DisplayTask` is executed for the last time it throws NPE here `target.getStage().addActor(fadeIn())` because target already has no stage.

Despite abnormal last `enter` event, I would suggest to use `Action` as base for `DisplayTask` and assign it to stage's root or target actor (first one is better).

**Comment by kotcrab** [comment]
Thanks for reporting. I can't reproduce this on LWJGL3 demo app running from soruce. I got expected events:
```
enter pointer -1
enter pointer 0 (clicking actor to remove it)
exit pointer 0
exit pointer -1
```
Events with non -1 pointers are ignored because they mean mouse clicks. Last exit cancels tasks.

> Despite abnormal last enter event, I would suggest to use Action as base for `DisplayTask` and assign it to stage's root or target actor (first one is better).

Are you sure that would help? `DisplayTask` would still NPE because target stage will be set to null already. Timer runs on different thread but it executres tasks on GDX thread. (it uses `Gdx.app.postRunnable(...)`). 

I just added simple check in for null stage in `DisplayTask`.
